external/kafka-0-10/src/test/scala/org/apache/spark/streaming/kafka010/KafkaTestUtils.scala

```diff
@@ -20,25 +20,24 @@ package org.apache.spark.streaming.kafka010
 import java.io.{File, IOException}
 import java.lang.{Integer => JInt}
 import java.net.InetSocketAddress
-import java.util.{Map => JMap, Properties}
 import java.util.concurrent.TimeoutException
-
-import scala.annotation.tailrec
-import scala.collection.JavaConverters._
-import scala.util.control.NonFatal
+import java.util.{Properties, Map => JMap}
```

Comment:
I moved the file from src/main to src/test because KafkaTestUtils does need the full kafka dependency (but is not used outside the tests).